qiskit/optimization/algorithms/admm_optimizer.py

```diff
@@ -53,6 +53,7 @@ def __init__(self,
                  tau_decr: float = 2,
                  mu_res: float = 10,
                  mu_merit: float = 1000,
+                 warm_start: Optional[bool] = False,
```

Comment:
```suggestion
                 warm_start: bool = False,
```
As this does not allow None this parameter is not Optional in the sense this is used in.